beetsplug/mbsync.py

```diff
@@ -82,18 +84,29 @@ def singletons(self, lib, query, move, pretend, write):
                                item_formatted)
                 continue
 
-            # Get the MusicBrainz recording info.
-            track_info = hooks.track_for_mbid(item.mb_trackid)
-            if not track_info:
-                self._log.info(u'Recording ID not found: {0} for track {0}',
-                               item.mb_trackid,
-                               item_formatted)
-                continue
+            valid_trackid = re.match(
+                     r"(\d|\w){8}-(\d|\w){4}-(\d|\w){4}-(\d|\w){4}-(\d|\w){12}",
```

Comment:
Perhaps this regex should go as a constant at the top of the file so it’s more easily identified.